src/parser/cds.lalrpop

```diff
@@ -156,7 +157,9 @@ Fields: Vec<Box<dyn ASTTerm>> = {
 }
 
 Field: Box<FieldTerm> = {
-    <name:Name> ":" <type_name:Name> ";" => FieldTerm::new_boxed(name, type_name)
+    <name:Name> ":" <type_name:Name> ";" => FieldTerm::new_boxed(name, type_name, None),
+    <name:Name> ":" <type_name:Name> "(" <type_arguments:TypeArguments> ")" ";" =>
+         FieldTerm::new_boxed(name, type_name, Some(type_arguments))
 }
```

Comment:
Could we use syntax errors handling from lalrpop just to improve durability?
For instance:
```
Num: i32 = {
    r"[0-9]+" =>? i32::from_str(<>)
        .map_err(|_| ParseError::User {
            error: "number is too big"
        })
};
```
[source](http://lalrpop.github.io/lalrpop/tutorial/007_fallible_actions.html)